ci/docker/runtime_functions.sh

```diff
@@ -187,13 +187,50 @@ build_dynamic_libmxnet() {
     ninja
 }
 
+build_onnx_and_onnx_tensorrt() {
+    set -ex
+    # Build ONNX
+    pushd .
+    echo "Installing ONNX."
+    cd 3rdparty/onnx-tensorrt/third_party/onnx
+    rm -rf build
+    mkdir -p build
+    cd build
+    cmake \
+        -DCMAKE_CXX_FLAGS=-I/usr/include/python${PYVER}\
```

Comment:
Suggest you delete this line instead of adding PYVER. Then it should be autodetected

Reply:
Good idea. Will do that.

Reply:
No luck for either way. @leezu How does ninja pick a  python version?

Reply:
@leezu  Is that OK I do alias python in the Dockerfile.build.jetson file? I saw only python3 was installed.

Reply:
I think the problem is that we're using a 2 years old version of onnx via the onnx tensorrt. That version seems to require Python 2. I suggest you update the submodule tensorrt onnx submodule to the latest release which hopefully supports Python 3.

This should be unrelated to ninja and above CMAKE_CXX_FLAGS

I would suggest not to introduce such symlink, as the upstream scripts may fail in subtle ways when switching from python2 to python3

Reply:
Make sense. I just did a commit of alias to give it a try. I will revoke that later.